test/integration/pods/pods_test.go

```diff
@@ -155,7 +157,9 @@ func TestPodReadOnlyFilesystem(t *testing.T) {
 	ns := framework.CreateTestingNamespace("pod-readonly-root", s, t)
 	defer framework.DeleteTestingNamespace(ns, s, t)
 
-	client := client.NewOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Default.GroupVersion()}})
```

Comment:
There is a "contact a human" button on the page: https://help.github.com/articles/unsubscribing-from-conversations/

My guess is that it is a github bug where you were mentioned previously and somehow that is overriding your mute every time.

(You can probably also mute the thread in gmail if you're using gmail)